test/DynamoCoreWpfTests/CoreUITests.cs

```diff
@@ -548,10 +567,10 @@ public void PreferenceSetting()
                 // First time run, check if dynamo did set it back to false after running
                 Assert.AreEqual(false, UsageReportingManager.Instance.FirstRun);
 
-                // CollectionInfoOption To TRUE
+                // CollectionInfoOption To FALSE
                 UsageReportingManager.Instance.SetUsageReportingAgreement(true);
```

Comment:
I'm confused by this change - it sets instrumentation to true? Then restarts and it's false? Can you explain why that is intended?

Reply:
Because `IsUsageReportingApproved` is dominated by the other setting
`IsUsageReportingApproved = approved && IsAnalyticsReportingApproved;`

So that functions are behaving same with UI

Reply:
thanks, would a comment in the code be possible?

Reply:
Will add now